src/modules/github/github/GitHub.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { setUserName, getRespo } from "./github.actions";
+
+const GitHub = () => {
+  const [form, setForm] = useState({
+    name: "",
+    repo: "",
+  });
+  const repos = useSelector((state) => state.github.repos);
+  const dispatch = useDispatch();
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    dispatch(setUserName(form.name));
```

Comment:
Zastanawiam się czy ten element jest niezbędny w magazynie, ale nie jest to błąd :)